daprdocs/content/en/getting-started/quickstarts/jobs-quickstart.md

```diff
@@ -0,0 +1,542 @@
+- [Latest version of Go](https://go.dev/dl/).
+<!-- IGNORE_LINKS -->
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+<!-- END_IGNORE -->
+
+### Step 2: Set up the environment
+
+Clone the [sample provided in the Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/jobs).
+
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+
+From the root of the Quickstarts directory, navigate into the jobs directory:
+
+```bash
+cd jobs/go/sdk
+```
+
+### Step 3: Schedule jobs
+
+In the terminal, run the `job-service` app:
+
+```bash
+dapr run --app-id job-service --app-port 6200 --dapr-http-port 6280 --dapr-grpc-port 6281 --app-protocol grpc -- go run .
+```
+
+**Expected output**
+
+```text
+== APP == dapr client initializing for: 127.0.0.1:6281
+== APP == Registered job handler for:  R2-D2
+== APP == Registered job handler for:  C-3PO
+== APP == Registered job handler for:  BB-8
+== APP == Starting server on port: 6200
+```
+
+In a new terminal window, run the `job-scheduler` app:
+
+```bash
+dapr run --app-id job-scheduler --app-port 6300 -- go run .
+```
+
+**Expected output**
+
+```text
+== APP == dapr client initializing for: 
+== APP == Get job response:  {"droid":"C-3PO","Task":"Memory Wipe"}
+== APP == Get job response:  {"droid":"BB-8","Task":"Internal Gyroscope Check"}
+== APP == Job deleted:  BB-8
+```
+
+Return to the `job-service` app terminal window. The output should be:
+
+```text
+== APP == Job scheduled:  R2-D2
+== APP == Job scheduled:  C-3PO
+== APP == 2024/07/17 18:25:36 job:{name:"C-3PO"  due_time:"10s"  data:{value:"{\"droid\":\"C-3PO\",\"Task\":\"Memory Wipe\"}"}}
+== APP == Job scheduled:  BB-8
+== APP == 2024/07/17 18:25:36 job:{name:"BB-8"  due_time:"15s"  data:{value:"{\"droid\":\"BB-8\",\"Task\":\"Internal Gyroscope Check\"}"}}
+== APP == Starting droid: R2-D2
+== APP == Executing maintenance job: Oil Change
+== APP == Starting droid: C-3PO
+== APP == Executing maintenance job: Memory Wipe
+```
+
+{{% /codetab %}}
+
+{{< /tabs >}}
+
+
+## Watch the demo
+
+See the jobs API in action using a Go HTTP example, recorded during the [Dapr Community Call #107(https://www.youtube.com/live/WHGOc7Ec_YQ?si=JlOlcJKkhRuhf5R1&t=849)].
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/WHGOc7Ec_YQ?si=JlOlcJKkhRuhf5R1&amp;start=849" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+## Tell us what you think!
+
+We're continuously working to improve our Quickstart examples and value your feedback. Did you find this Quickstart helpful? Do you have suggestions for improvement?
+
+Join the discussion in our [discord channel](https://discord.com/channels/778680217417809931/953427615916638238).
+
+## Next steps
+
+- HTTP samples of this quickstart:
+  - [Go](todo)
```

Comment:
Can this be completed yet?

Reply:
@msfussell this is already merged with the quickstarts 1.14 branch.